addons/web/static/src/model/relational_model/utils.js

```diff
@@ -323,12 +323,13 @@ export function getFieldDomain(record, fieldName, domain) {
 }
 
 export function getBasicEvalContext(config) {
-    const { uid, allowed_company_ids } = config.context;
+    const { uid, allowed_company_ids, current_company_country_code } = config.context;
     return {
         context: config.context,
         uid,
         allowed_company_ids,
-        current_company_id: config.currentCompanyId,
+        current_company_id: config.currentCompanyId, // This is realy needed ?
```

Comment:
yes, grepping "current_company_id"  makes some fields appear with that special key in some domain for some fields;
```py
project_id = fields.Many2one(domain="['|', ('company_id', '=', False), '&', ('company_id', '=?', company_id), ('company_id', '=', current_company_id), ('allow_billable', '=', True), '|', ('pricing_type', '=', 'task_rate'), ('is_fsm', '=', True), ('allow_timesheets', 'in', [service_policy == 'delivered_timesheet', True])]")
```

Reply:
It's not used in the views.